clients/src/main/resources/common/message/StreamsHeartbeatRequest.json

```diff
@@ -71,9 +71,10 @@
           "about": "The cumulative lag for the task" }
       ]
     },
+    { "name": "UserData", "type": "bytes", "versions": "0+", "nullableVersions": "0+", "default": "null",
+      "about": "Opaque user data to be passed to the client-side assignor. Null if unchanged since last heartbeat." },
```

Comment:
This is a request by Sophie. I think we want to avoid them for the server-side assignors, but we can allow them for client-side assignors if it makes Responsive happy. It also allows them to trigger the client-side assignor (changing the user data will cause the member data to be updated, triggering an assignor call).

Reply:
Hm, I am not convinced. Let me think about this.